words/predicates.txt

```diff
@@ -164,9 +199,12 @@ changeable
 charm
 charmed
 charming
+chartreuse
 chatter
 checker
 checkered
+cheddar
+cheeky
```

Comment:
I'm not sure if this is a reason to remove, but this is often used for describing lingerie.